packages/cli/src/commands/auth/login.ts

```diff
@@ -0,0 +1,28 @@
+import { AbstractCommand } from '../../abstract.command'
+import { Command, Param } from '../../decorators'
+import chalk from 'chalk'
+
+@Command({
+  name: 'login',
+  description: 'test command for logging in via reapit connect',
+})
+export class LoginCommand extends AbstractCommand {
+  async run(
+    @Param({
+      name: 'clientId',
+      description: 'clientId of your app',
+      required: true,
+    })
+    clientId: string,
```

Comment:
Is the goal to log in to the CLI so that we can do away with API keys? if so maybe the CLI should have its own client ID rather than using theirs?